include/translate-option.php

```diff
@@ -404,6 +404,7 @@ protected function check_value_recursive( $old_values, $values, $key, $mo ) {
 	 * @return string Sanitized value.
 	 */
 	public function sanitize_option( $value, $name ) {
-		return sanitize_option( $name, $value );
+		$value = sanitize_option( $name, $value );
+		return is_scalar( $value ) ? (string) $value : '';
```

Comment:
I am in favor to keep the code as it was and handle this with doc since it's more a doc issue, as `sanitize_option()` doesn't change the type of `$value`.